GameEngine/Lang/en.php

```diff
@@ -130,6 +130,7 @@
 define("CROP_COM",CROP." consumption");
 define("PER_HR","per hour");
 define("PROD_HEADER","Production");
+define("CAPITAL_NAME","Capital");
```

Comment:
I'd use the name `CAPITAL` rather than `CAPITAL_NAME` which makes think about the name the player used for their capital.

Reply:
The name `CAPITAL` is already being used! 
`define("CAPITAL","This is your capital");`

Reply:
@andreapavoni 
Is it possible to use a different variable name? 
For example `CAPITAL_HEADER` ?

Reply:
Or 
`CAPITAL_TITLE` ,
`CAPITAL_LABEL` ?

Reply:
@andreapavoni 
Sent a new commit .